### ISSUE TITLE
indicate why `javascript:` or `places:` bookmarks don't open into container tabs

### ISSUE THREAD

**Issue by groovecoder** [mozilla/multi-account-containers#1562]
### Steps to reproduce
1. Have a bookmark that starts with `javascript:` or `places:`
2. Enable bookmark menus in Multi-Account Containers preferences
3. Right-click a bookmark from step 1
4. Select Open Bookmark in Container Tab -> Personal (for example)

### Actual behavior
Nothing happens

### Expected behavior
Should be some indicator why `javascript:` or `places:` urls don't open into container tabs

### Notes
https://github.com/mozilla/multi-account-containers/pull/1537/files/7c08cf31901c0ccfbecbffdbc6526c44e86a4305#r339101369

**Comment by yellowwoods12** [comment]
https://github.com/mozilla/multi-account-containers/pull/1537#discussion_r341803244
Hi @groovecoder, I guess the above solution will solve this. What do you think?

**Comment by Rob--W** [comment]
It's not just `javascript:`. If the user has bookmarked anything that extensions cannot open (e.g. `about:preferences`, then nothing would happen.

In my extension (that served as the inspiration for this check), I just filtered `javascript:` and `place:` (note: not `places:`) because they are not uncommon.
https://github.com/Rob--W/bookmark-container-tab/blob/1fe029d5b0fa90b42b237bd7926688c48268ce57/background.js#L137-L140

**Comment by yellowwoods12** [comment]
> It's not just `javascript:`. If the user has bookmarked anything that extensions cannot open (e.g. `about:preferences`, then nothing would happen.
> 
> In my extension (that served as the inspiration for this check), I just filtered `javascript:` and `place:` (note: not `places:`) because they are not uncommon.
> https://github.com/Rob--W/bookmark-container-tab/blob/1fe029d5b0fa90b42b237bd7926688c48268ce57/background.js#L137-L140

Yes, exactly. As a solution we can generate an alert message everytime an error message is returned as a promise during the new tab creation. I think this is one feasible and easy solution for now. 
@Rob--W what do you think about this?

**Comment by Rob--W** [comment]
That could be annoying.

`place:` shouldn't be opened, it doesn't have a meaning outside of bookmarks.
`javascript:` doesn't have meaning for new tabs, they're typically used to run on existing sites.

For other URLs, you could consider opening an extension page with a link to them (so that the user can click on the link itself). However, I think that it is such a rare edge case that it's not worth implementing. After all, a user can just open the link via the usual ways (without containers; containers are mainly useful for http(s) sites in practice).

**Comment by yellowwoods12** [comment]
> That could be annoying.
> 
> `place:` shouldn't be opened, it doesn't have a meaning outside of bookmarks.
> `javascript:` doesn't have meaning for new tabs, they're typically used to run on existing sites.
> 
> For other URLs, you could consider opening an extension page with a link to them (so that the user can click on the link itself). However, I think that it is such a rare edge case that it's not worth implementing. After all, a user can just open the link via the usual ways (without containers; containers are mainly useful for http(s) sites in practice).

I agree that it is a rare edge case so it depends on the team now whether they want to implement it or not.
